.env_SAMPLE

```diff
@@ -36,9 +36,10 @@ export DJANGO_HTTP_PORT=8000
 #export EXTERNAL_LINK_CSS=<external_links_css_class_name>
 export ALLOW_ADMIN_URL=True
 #export ENABLE_AKAMAI_CACHE_PURGE=<set_to_enable_purging_akamai_cache>
-#export AKAMAI_USER=<akamai_username>
-#export AKAMAI_PASSWORD=<akamai_password>
-#export AKAMAI_OBJECT_ID=<akamai_object_id>
+#export AKAMAI_CLIENT_TOKEN=<akamai_client_token>
+#export AKAMAI_CLIENT_SECRET=<akamai_client_secret>
+#export AKAMAI_ACCESS_TOKEN=<akamai_access_token>
+#export AKAMAI_FAST_PURGE_URL=<akamai_fast_purge_url>
```

Comment:
We'll need to ensure we make the appropriate changes to the variables in `ansible-consumer-finance/provisioning/group_vars`, and to the `ansible-consumer-finance-frontend/templates/configs/environment.json.j2` template file.